samples/attested_tls/server/enc/openssl_server.cpp

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <arpa/inet.h>
+#include <openenclave/enclave.h>
+#include <openssl/evp.h>
+#include <openssl/ssl.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include "../../common/openssl_utility.h"
+
+extern "C"
+{
+    int set_up_tls_server(char* server_port, bool keep_server_up);
+};
+
+int verify_callback(int preverify_ok, X509_STORE_CTX* ctx);
+
+int initalize_ssl_context(SSL_CTX*& ctx)
+{
+    int ret = -1;
+    if ((ctx = SSL_CTX_new(TLS_server_method())) == NULL)
```

Comment:
Use `nullptr` instead (here and the rest of the file).

Reply:
Or rename file from .cpp to .c to use C instead of C++